packages/app/obojobo-repository/shared/components/repository-nav.jsx

```diff
@@ -1,12 +1,15 @@
 require('./repository-nav.scss')
 
 const React = require('react')
-const { useState } = require('react')
 const Avatar = require('./avatar')
+const Notification = require('./notification')
 
 const RepositoryNav = props => {
 	let timeOutId
-	const [isMenuOpen, setMenuOpen] = useState(false)
+	const [isMenuOpen, setMenuOpen] = React.useState(false)
+	const [isNotificationsOpen, setNotificationsOpen] = React.useState(false) // New state for notification
+	const [notificationsExist, setNotificationsExist] = React.useState(false)
```

Comment:
There doesn't appear to be anything in the file using `setNotificationsExist`. It also looks like the notification indicator remains on the screen even if all notifications are dismissed.

Consider passing `setNotificationsExist` into the `Notification` component and adding some logic to that component to call it with `false` if all notifications are dismissed.